sites/website/src/docs/design-systems/fast-frame.md

```diff
@@ -132,21 +132,27 @@ provideFASTDesignSystem()
     )
 ```
 
-It's also worth noting that this can be used to simply extend the existing styles, by importing the originals and composing those with new styles. Here's what that would look like:
+It's also worth noting that this can be used to extend the existing styles, by importing the originals and composing those with new styles by calling the style function. Here's what that would look like:
 
 ```ts
 provideFASTDesignSystem()
     .register(
         fastButton({
-            styles: css`
-                /* import and compose the original styles */
-                ${buttonStyles}
+            styles: (ctx, def) => css`
+                ${buttonStyles(ctx, def)}
                 /* add your style augmentations here */
             `
         })
     )
 ```
 
+:::important
+
+At present, there is a minor typing bug across all the style and template functions, so you will need to cast the second argument as follows `${buttonStyles(ctx, def as any)}`. This issue will be fixed in an upcoming release.
```

Comment:
Can we add a tracking issue to this so folks can know the status and so remember to remove this when it's merged?

Reply:
Good call. Writing issue now, and then will update this with the link.

Reply:
Done!